cirq-google/cirq_google/devices/grid_device.py

```diff
@@ -112,12 +112,23 @@ class _GateRepresentations:
     ),
     _GateRepresentations(
         gate_spec_name='phased_xz',
-        deserialized_forms=[cirq.PhasedXZGate, cirq.XPowGate, cirq.YPowGate, cirq.PhasedXPowGate],
+        deserialized_forms=[
+            cirq.PhasedXZGate,
+            cirq.XPowGate,
+            cirq.YPowGate,
+            cirq.PhasedXPowGate,
+            cirq.HPowGate,
+            cirq.IdentityGate,
+            cirq.ops.SingleQubitCliffordGate,
+        ],
         serializable_forms=[
+            cirq.GateFamily(cirq.IdentityGate),
```

Comment:
This (and the one in `deserialized_forms` should be `cirq.I` instead of `cirq.IdentityGate`, since the latter is for any number of qubits.

Could you also leave a TODO for generalized `cirq.IdentityGate` support? There's an open question of whether we want to serialize identity gates, and if not, whether all devices should support it by default.

Reply:
done